common/SC_PaUtils.hpp

```diff
@@ -0,0 +1,167 @@
+// helper functions for SuperCollider server's PortAudio backend
+
+#ifndef SC_PAUTILS_HPP
+#define SC_PAUTILS_HPP
+
+#include "portaudio.h"
+#include <stdlib.h>
```

Comment:
please use the C++ header names (`<cstdlib>`), also why do you include `<cstdlib>`? You should also be including `<cstring>` which is [where `std::strstr` is defined](https://en.cppreference.com/w/cpp/string/byte/strstr), and `<cstdio>` where `std::fprintf` is defined.

also include `<string>` since you use std::string below

but, you only need to include `<string>` in this header when you move the definitions to a cpp file